.pre-commit-config.yaml

```diff
@@ -1,28 +1,37 @@
 repos:
   - repo: https://github.com/ambv/black
-    rev: 22.12.0
+    rev: 23.7.0
     hooks:
       - id: black
   - repo: https://github.com/pycqa/isort
     rev: 5.12.0
     hooks:
       - id: isort
         args:
-        - "--profile=black"
         - "--filter-files"
         - "--project=autora"
   - repo: https://github.com/pycqa/flake8
-    rev: 6.0.0
+    rev: 6.1.0
     hooks:
       - id: flake8
         args:
         - "--max-line-length=100"
         - "--extend-ignore=E203"
+        - "--per-file-ignores=__init__.py:F401"
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: "v0.991"
+    rev: "v1.5.1"
     hooks:
       - id: mypy
-        additional_dependencies: [types-requests]
+        additional_dependencies: [types-requests,scipy,pytest]
```

Comment:
@benwandrew have you tried including the additional types here?

Reply:
resolved by adding relevant overrides

`[[tool.mypy.overrides]]
module = ["matplotlib.*", "sklearn.*", "pandas.*", "numpy.*","scipy.*"]
ignore_missing_imports=true`